docs/tutorial/examples/clip.py

```diff
@@ -0,0 +1,14 @@
+from onnxscript import opset15 as op
+from onnxscript import script
+
+
+@script()
+def Clip(input, min, max):
```

Comment:
Please add test to cover this function. Or just enable the test at https://github.com/microsoft/onnx-script/blob/b2fd55bd7d8a22383c15127ffaeb074756ac3563/onnxscript/tests/functions/onnxfns1A_test.py#L75